examples/golang-pgo/docker-compose.yml

```
@@ -1,15 +1,23 @@
-version: "3"
+version: '3'
 services:
   rideshare-go:
     environment:
-      - REGION=us-east
-      - PYROSCOPE_SERVER_ADDRESS=http://pyroscope:4040
+    - REGION=us-east
+    - PYROSCOPE_SERVER_ADDRESS=http://pyroscope:4040
     build:
       context: .
     ports:
-      - '5001:5001'
-
+    - 5001:5001
   pyroscope:
     image: grafana/pyroscope:latest
     ports:
-      - '4040:4040'
+    - 4040:4040
+  grafana:
+    image: grafana/grafana:latest
+    environment:
+    - GF_INSTALL_PLUGINS=grafana-pyroscope-app
+    - GF_AUTH_ANONYMOUS_ENABLED=true
```

Any particular reason why this is not there? I think at least `GF_AUTH_ANONYMOUS_ORG_ROLE=Admin` is need otherwise Explore is not shown:

```suggestion
    - GF_INSTALL_PLUGINS=grafana-pyroscope-app
    - GF_AUTH_ANONYMOUS_ENABLED=true
    - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
    - GF_AUTH_DISABLE_LOGIN_FORM=true
 ```